sdk/src/main/scala/com/horizen/forge/package.scala

```diff
@@ -0,0 +1,8 @@
+package com.horizen
+
+import scorex.core.NodeViewHolder.CurrentView
+
+package object forge {
+  type View = CurrentView[SidechainHistory, SidechainState, SidechainWallet, SidechainMemoryPool]
```

Comment:
Probably we can define `SidechainView[SidechainHistory, ...]` and `SidechainNodeView[SidechainNodeHistory, ...]` types/case classes inside `SidechainNodeViewHolder` object and later reuse it in `Forger`, `API`, `CertificateSubmitter` classes and so on.
Now it can be used in Forger, later other places can be refactored as well.